lmms_eval/tasks/video-tt/videott_no_leading_oe.yaml

```
@@ -0,0 +1,42 @@
+    higher_is_better: true
+lmms_eval_specific_kwargs:
+  default:
+    pre_prompt: ""
+    post_prompt: ""
+  gpt4v:
+    pre_prompt: ""
+    post_prompt: ""
+  llava_vid:
+    pre_prompt: ""
+    post_prompt: ""
+  # qwen_vl:  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)